### DISCUSSION TITLE
init_bulk meet RuntimeError: Meet errors will handle unexpected submission state.

### DISCUSSION THREAD

**Discussion by Ziyang-Qiu** [deepmodeling/dpgen#1215] [Q&A]
The error
![图片](https://github.com/deepmodeling/dpgen/assets/127955043/a016961d-0a3f-4a45-86a0-b952c1b0ae6e)

The fp.log 
![图片](https://github.com/deepmodeling/dpgen/assets/127955043/f88b05ac-15ad-4386-a03c-76e0d4efe0dc)

My init.json
![图片](https://github.com/deepmodeling/dpgen/assets/127955043/471b94e4-d043-4b75-9070-221d8d7c0cee)

Thanks a lot

**Comment by njzjz** [comment]
It seems to me that the segmentation fault error comes from VASP. If the generated input files have nothing wrong, you might need to contact VASP support.
